netlify-functions/src/main/scala/feral/netlify/facade/Context.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feral.netlify.facade
+
+import scala.scalajs.js
+import scala.scalajs.js._
+
+@js.native
+private[netlify] sealed trait Context extends js.Object {
+  def callbackWaitsForEmptyEventLoop: Boolean = js.native
+  def functionName: String = js.native
+  def functionVersion: String = js.native
+  def invokedFunctionArn: String = js.native
+  def memoryLimitInMB: Int = js.native
```

Comment:
This is a `String` in the netlify TS, any ideas about that?
https://github.com/netlify/functions/blob/main/src/function/context.ts#L10

Reply:
yeah I saw that, but if you try running it locally you'll get a number. I suspect TS doesn't care as there's no runtime, and coerces the value to a string, but scalajs sees a number and freaks out?

Reply:
Cool, so it's a mistake with the TS. Good enough for me.

Reply:
Reported upstream https://github.com/netlify/functions/pull/251